doc/cephfs/fs-volumes.rst

```diff
@@ -214,10 +220,19 @@ Subvolumes can be created by cloning subvolume snapshots. Cloning is an asynchro
 data from a snapshot to a subvolume. Due to this bulk copy nature, cloning is currently inefficient for very huge
 data sets.
 
-Before starting a clone operation, the snapshot should be protected. Protecting a snapshot ensures that the snapshot
-cannot be deleted when a clone operation is in progress. Snapshots can be protected using::
+.. note:: Removing a snapshot (source subvolume) would fail if there are pending or in progress clone operations.
+
+Protecting snapshots prior to cloning was a pre-requisite in the Nautilus release, and the commands to protect/unprotect
+snapshots were introduced for this purpose. This pre-requisite, and hence the commands to protect/unprotect, is being
+deprecated in mainline CephFS, and may be removed from a future release.
 
+The commands being deprecated are:
```

Comment:
Commands below are not rendered right in [master](https://docs.ceph.com/docs/master/cephfs/fs-volumes/#cloning-snapshots) version of the change, they need an additional space. Should this be addressed here amending the patch (and noting it in the conflicts)?

Reply:
Fixed adding the required line space and also mentioned the same in the conflicts section of the commit message.